nixos/modules/services/hardware/kanata.nix

```diff
@@ -47,6 +47,16 @@ let
           See <link xlink:href="https://github.com/jtroo/kanata"/> for more information.
         '';
       };
+      extraArgs = mkOption {
+        type = types.str;
+        default = "";
+        description = ''
+          Extra command line arguments passed to kanata.
+
+          If the port flag is used, you need to change some network-related
+          hardening options of this systemd service using lib.mkForce.
```

Comment:
That is really vague. I wouldn't know what I would need to change here.

Reply:
Which one do you think is better?
- provide an option `port`. if `port` is not `null`, change needed hardening options
- list hardening options needing to be changed in the description

Reply:
> * list hardening options needing to be changed in the description

Thats not really the nixos style.



> * provide an option `port`. if `port` is not `null`, change needed hardening options

that sounds way better.